job-service/src/main/java/com/hpe/caf/services/job/api/JobsPut.java

```diff
@@ -205,10 +207,10 @@ public static String createOrUpdateJob(final String partitionId, String jobId, N
             //  Get database helper instance.
             try {
                 QueueServices queueServices = QueueServicesFactory.create(config, jobTask.getTaskPipe(),codec);
-                LOG.debug("createOrUpdateJob: Sending task data to the target queue...");
```

Comment:
Is moving this debug line down intentional?

Reply:
Oh, no it wasn't. I'll fix it now.